Formula/llvm.rb

```diff
@@ -284,16 +274,18 @@ def install
       system "make", "install"
     end
 
-    if build.with? "clang"
-      (share/"clang/tools").install Dir["tools/clang/tools/scan-{build,view}"]
-      inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
-      bin.install_symlink share/"clang/tools/scan-build/bin/scan-build", share/"clang/tools/scan-view/bin/scan-view"
-      man1.install_symlink share/"clang/tools/scan-build/man/scan-build.1"
-    end
+    (share/"clang/tools").install Dir["tools/clang/tools/scan-{build,view}"]
+    inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
+    bin.install_symlink share/"clang/tools/scan-build/bin/scan-build", share/"clang/tools/scan-view/bin/scan-view"
+    man1.install_symlink share/"clang/tools/scan-build/man/scan-build.1"
+
+    # Create symlinks for compatibility with prior "clang-omp" formula.
+    bin.install_symlink bin/"clang" => "clang-omp"
+    bin.install_symlink bin/"clang" => "clang-omp++"
```

Comment:
I didn't know if I was worth preserving the links temporarily? But I guess since this formula is `keg_only` and `clang-omp` wasn't we wouldn't automatically be saving any scripts that had come to use that symlink. Happy to remove.